crates/sui-types/src/gas_model/gas_v1.rs

```diff
@@ -237,15 +237,15 @@ fn to_internal(external_units: GasUnits) -> InternalGas {
 
 impl SuiGasStatus {
     fn new(
-        move_gas_status: GasStatus,
+        move_gas_status_mut: GasStatus,
```

Comment:
Not sure about this rename? Thoughts on keeping it as it was?

Reply:
Indeed a mistake. Will evert